### PR TITLE
Add macro support

### PR DESCRIPTION
**Pull request by lthorne1**
CloudFormation supports custom macros, this PR allows the AUTO_EXPAND capability so we can leverage those.

### CODE REVIEW

**Comment by steven-cuthill-otm** [comment]
can someone review this ? maybe @steveataws  ?

**Comment by steveataws** [comment]
All looks good to me; I will reach out to the team that now owns the tools to get this merged and to see if they can give some indication of when the next release may be.

**Comment by steven-cuthill-otm** [comment]
bump !

**Comment by jcwhisman** [comment]
@steveataws is there an update on this? This is preventing us from adding some new CFN templates to our CI/CD pipeline.

File: Tasks/CloudFormationCreateOrUpdateStack/task.json [hunk 1]
```diff
@@ -224,6 +224,15 @@
             "groupName": "Capabilities",
             "helpMarkDown": "Capabilities that must be specified before AWS CloudFormation can update certain stacks. Some stack templates might include resources that can affect permissions in your AWS account, for example, by creating new AWS Identity and Access Management (IAM) users. For those stacks, you must explicitly acknowledge their capabilities by specifying this parameter.\n\nIf your stack manipulates IAM resources with custom names, you must add this capability otherwise an InsufficientCapabilities error will be returned."
         },
+        {
+            "name": "capabilityAutoExpand",
+            "type": "boolean",
+            "label": "Allow use of CloudFormation Macros ('CAPABILITY_AUTO_EXPAND')",
+            "defaultValue": "true",
```
Comment (hunterwerlla) [inline Tasks/CloudFormationCreateOrUpdateStack/task.json]: Thanks for the contribution!

The only thing I am worried about is this is set to true by default, I'll have to check if this is an issue or not.

Comment (lthorne1) [reply Tasks/CloudFormationCreateOrUpdateStack/task.json]: Would changing it's default to false allow this to be accepted?
```suggestion
            "defaultValue": "false",
```

**Pull request merged**